src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,80 @@
 import React from 'react';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  selectedTodoUser: User | null;
```

Comment:
```suggestion
  onSelect: User | null;
```

Reply:
I've shortened selectedTodoUser to selectedUser. I think onSelect would fit if it was an event callback and I have a user object here.

Reply:
sorry, my bad here